iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergOutputCommitter.java

```diff
@@ -97,6 +100,11 @@ public class HiveIcebergOutputCommitter extends OutputCommitter {
   private static final String FOR_COMMIT_EXTENSION = ".forCommit";
 
   private static final Logger LOG = LoggerFactory.getLogger(HiveIcebergOutputCommitter.class);
+  private static final HiveIcebergOutputCommitter hiveIcebergOutputCommitter = new HiveIcebergOutputCommitter();
```

Comment:
maybe simply `outputCommitter`, it's in iceberg class already

Reply:
Done.